package.json

```diff
@@ -2,11 +2,17 @@
   "name": "@mongoosejs/in-memory-driver",
   "version": "0.0.1",
   "scripts": {
-    "test": "mocha test/*.test.js"
+    "test": "mocha test/*.test.js",
+    "lint": "eslint ."
   },
   "devDependencies": {
+    "@typescript-eslint/eslint-plugin": "^5.30.7",
```

Comment:
Let's avoid adding TypeScript packages for now. If we add an `index.d.ts` for this project, it will probably be a one-liner anyway. Not worth the extra overhead right now.

Reply:
removed in f095711fa570a587e4f367afa33a0c8d9c3408c2, if this should be merged without squash, this commit can be reverted again to have all typescript dependencies again with eslint configuration